vueapp/src/App.vue

```diff
@@ -78,13 +78,31 @@ export default {
     ...mapActions({        
       toggleSettingsDialogVisibility: 'settings/toggleSettingsDialogVisibility',
       toggleEditMode: 'tilePad/toggleEditMode'
-    })
+    }),
+    populateVoiceData (windowVoices){
+      const voiceOptions = windowVoices.map((voice, index) => {
+        return { text: `${voice.name} (${voice.lang})`, value: index };
+      }).sort((a, b) => a.text.localeCompare(b.text));
+
+      this.$store.dispatch('settings/setVoices', windowVoices);

```

Comment:
Could you change these 2 to being Called from map actions please? 

Also I do not believe this is actually storing the SpeechSynthesisVoice object in vuex. I believe it just stores a bunch of empty objects when I checked, and looks like Tile.Vue just calls the `getVoices()` to populate the voice to use. Does store voiceOptions nicely though!